tooling/cli/lib/parse-translations.js

```diff
@@ -58,49 +58,69 @@ const KEYS_WITH_PLURALS = [];
       }
     }
   }
+  return FINAL_TRANSLATIONS;
 
-  await writeFile(
-    path.join(OP_DIR, 'locales/translations.json'),
-    JSON.stringify(FINAL_TRANSLATIONS, null, 2)
-  );
-})()
-  .then(() => {
-    console.log('>>> Translations parsed');
-    process.exit(0);
-  })
-  .catch(err => {
-    console.error('>>> Error parsing translations', err.message);
-    process.exit(1);
-  });
-
-function addToTranslations(key) {
-  const translatedLanguages = Object.keys(TI_TRANSLATIONS);
-  for (const lang of translatedLanguages) {
-    if (!FINAL_TRANSLATIONS[lang]) {
-      FINAL_TRANSLATIONS[lang] = { lms: {} };
-    }
+  function addToTranslations(key) {
```

Comment:
Think it might be easier to follow if you bump these up closer to the top of `processTranslations`, before they're called, since they're scoped to this function.